core/web/evm_transfer_controller_test.go

```diff
@@ -41,8 +41,9 @@ func TestTransfersController_CreateSuccess_From(t *testing.T) {
 	balance, err := assets.NewEthValueS("200")
 	require.NoError(t, err)
 
-	ethClient.On("PendingNonceAt", mock.Anything, key.Address).Return(uint64(1), nil)
+	ethClient.On("PendingNonceAt", mock.Anything, key.Address).Return(uint64(1), nil).Maybe()
```

Comment:
Why maybe instead of not at all?

Reply:
It is needed in [SyncOnChain](https://github.com/smartcontractkit/chainlink/blob/develop/core/chains/evm/txmgr/nonce_tracker.go#L120). It's [not always called](https://github.com/smartcontractkit/chainlink/blob/bfd9a3ffe11484f3dc413bb77a709f2c4bb281b0/core/chains/evm/txmgr/nonce_tracker.go#L105), that's why the `Maybe()`